### PR TITLE
How about this

### PR DESCRIPTION
**Pull request by jcubbage**
Basically same as @jSylvestre  suggested fix.

### CODE REVIEW

**Comment by jcubbage** [comment]
image

File: Keas.Mvc/Services/IdentityService.cs [hunk 1]
```diff
@@ -199,6 +225,10 @@ public async Task<string> BulkLoadPeople(string ppsCode, string teamslug)
                     {
                         var personResult = await GetOrCreatePersonFromKerberos(kerbResults.ResponseData.Results[0].UserId, team.Id);
                         newpeople += personResult.peopleCount;
+                        if (personResult.Person == null)
+                        {
+                            warning.Append($"IAM ID {id.IamId} failed to save.");
```
Comment (jSylvestre) [inline Keas.Mvc/Services/IdentityService.cs]: @jcubbage IAM Id isn't supper useful to me, so I'd add kerb as well, so I'd change it to:
warning.Append($"IAM ID: {id.IamId}, Kerb Id: {kerbResults.ResponseData.Results.First().UserId}  failed to save.");

File: Keas.Mvc/Services/IdentityService.cs [hunk 2]
```diff
@@ -199,6 +225,10 @@ public async Task<string> BulkLoadPeople(string ppsCode, string teamslug)
...
+                        }
                     }
```
Comment (jSylvestre) [inline Keas.Mvc/Services/IdentityService.cs]: @jcubbage Add an else with a warning here, but you can probably only have the IAM id as the kerb may not have been found. This should be the else to the `if (kerbResults.ResponseData.Results.Length > 0 && contactResult.ResponseData.Results.Length > 0)`

Comment (jSylvestre) [reply Keas.Mvc/Services/IdentityService.cs]: @jcubbage That will not catch IAM users that PPS returns, but that we don't find.
You would need it on the else of this if (If we want to report these)
`                    var kerbResults = await clientws.Kerberos.Search(KerberosSearchField.iamId, id.IamId);
                    var contactResult = await clientws.Contacts.Get(id.IamId);

                    if (kerbResults.ResponseData.Results.Length > 0 && contactResult.ResponseData.Results.Length > 0)
                    {
                        var personResult = await GetOrCreatePersonFromKerberos(kerbResults.ResponseData.Results[0].UserId, team.Id);
                        newpeople += personResult.peopleCount;
                        if (personResult.Person == null)
                        {
                            if (kerbResults.ResponseData.Results.Length > 0)
                            {
                                warning.Append($" IAM ID: {id.IamId}, Kerb Id: {kerbResults.ResponseData.Results.First().UserId} failed to save.");
                            }
                            else
                            {
                                warning.Append($" IAM ID {id.IamId} failed to save.");
                            }
                        }
                    }`

**Pull request merged**